CMakeLists.txt

```diff
@@ -118,6 +118,7 @@ adios_option(MPI       "Enable support for MPI" AUTO)
 adios_option(DataMan   "Enable support for DataMan" AUTO)
 adios_option(DataSpaces "Enable support for DATASPACES" AUTO)
 adios_option(SSC       "Enable support for SSC" AUTO)
+adios_option(Table     "Enable support for Table" AUTO)
```

Comment:
It looks like the engine isn't actually optional, i.e. there's no way to disable it, so this shouldn't be a user-facing option.

Reply:
I just checked and I am actually able to disable it by specifying 
cmake ~/d/a2table -DADIOS2_USE_Table=OFF

Reply:
@chuckatkins By the way, how can I quickly check if newer versions of nlohmann::json is already compatible with MSVC or PGI so I can update the library to have these compilers supported for some of my engines that are using nlohmann::json?

Reply:
> I just checked and I am actually able to disable it by specifying
> cmake ~/d/a2table -DADIOS2_USE_Table=OFF

I must have misread the logic.  Sorry about that.